make-rootfs.sh

```diff
@@ -77,6 +77,17 @@ function copy_services() {
     ln -sf /lib/systemd/system/rauc.service \
         ${ROOTFS_DIR}/etc/systemd/system/multi-user.target.wants/rauc.service
 
+    install -m 0644 ${SERVICES_DIR}/usb-rauc-install.service \
+        ${ROOTFS_DIR}/lib/systemd/system
+
+}
+
+function copy_usb_handlers_scripts(){
+    
```

Comment:
Everything in etc is automatically copied :)

Reply:
Yes, I did see that part, but I wanted to explicitly have that function because I also want to modify the permissions on the files. Maybe I should only do the chmod instead of install?